views/demo.pug

```diff
@@ -4,13 +4,13 @@ block content
 	.container-fluid
 		.row
 			.col-sm-3.col-lg-2
-				nav.side-navbar.navbar-default.navbar-fixed-side
+				.side-navbar
 					ul
-						li
-							a(href="#") Link-1
-						li
-							a(href="#") Link-2
-						li
-							a(href="#") Link-3
+						a(href="#")
+							li Link-1
+						a(href="#")
```

Comment:
I was wrapping  `<li>` inside `<a>` because it provides an ease of usability i:e a user can also click nearby the text to access the link which is maybe easy to use than just clicking on the text to access the link. 
And yes, I'll make classes in my next commit.